README.md

```diff
@@ -66,7 +66,7 @@ TPP REST APIs but please be advised this goes against Venafi recommendations.
 
 ## Prerequisites for using with Venafi as a Service
 
-1. The Venafi as a Service REST API is accessible at https://api.venafi.cloud from the system where VCert
+1. The Venafi as a Service REST API is accessible at https://api.venafi.cloud or https://api.eu.venafi.cloud(if you have an EU account) from the system where VCert
```

Comment:
Resolve if you are sure we need to use `api.eu.venafi.cloud` (as opposed to `api.venafi.eu`).

Reply:
Confirmed that the `api.venafi.eu` should be used and changed to use it